### PR TITLE
Milestone 6 : Create database based on diagram

### PR DESCRIPTION
**Pull request by Favourezeugwa**
## In this milestone, I and my partner;
- Created a file named `schema_based_on_diagram.sql` to implement the database from the diagram.
- Added Join tables from many-to-many relationships.
- Added the FK indexes

### CODE REVIEW

File: schema_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  PRIMARY KEY(id)
+);
+
+-- medical history
+CREATE TABLE medical_histories (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id INT NOT NULL,
+  status VARCHAR(250) NOT NULL,
+  FOREIGN KEY (patient_id) REFERENCES species(id);
```
Comment (zieeco) [inline schema_diagram.sql]: - Hey, I think you meant the `patients(id)` as the `species table` does not exist in the diagram given. please, kindly verify the table name to fix the issue. :pray:

File: schema_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+-- invoices
+CREATE TABLE invoices (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+  total_amount FLOAT(2) NOT NULL,
+  generated_at TIMESTAMP NOT NULL,
+  payed_at TIMESTAMP NOT NULL,
+  medical_history_id INT NOT NULL,
+  PRIMARY KEY (id),
+  FOREIGN KEY (medical_history_id) REFERENCES medical_history(id)
```
Comment (zieeco) [inline schema_diagram.sql]: - Hey, please could you verify the table name `medical_history` ? I supposed you meant `medical_histories`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Favourezeugwa,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice following the diagram. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_diagram.sql (hunk 2 above)
Comment (Favourezeugwa) [reply schema_diagram.sql]: Hi @zieeco thank you very much for your observations, I will make the adjustments for sure.

Comment (zieeco) [reply schema_diagram.sql]: @Favourezeugwa I'm glad that helped a little, you can reach out on slack if you need help.

File: schema_diagram.sql [hunk 3]
```diff
@@ -34,7 +34,7 @@ CREATE TABLE invoices (
   payed_at TIMESTAMP NOT NULL,
   medical_history_id INT NOT NULL,
   PRIMARY KEY (id),
-  FOREIGN KEY (medical_history_id) REFERENCES medical_history(id)
+  FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
```
Comment (mateo951) [inline schema_diagram.sql:37]: Addressed 😎

File: schema_diagram.sql [hunk 4]
```diff
@@ -14,7 +14,7 @@ CREATE TABLE medical_histories (
   admitted_at TIMESTAMP NOT NULL,
   patient_id INT NOT NULL,
   status VARCHAR(250) NOT NULL,
-  FOREIGN KEY (patient_id) REFERENCES species(id);
+  FOREIGN KEY (patient_id) REFERENCES patients(id);
```
Comment (mateo951) [inline schema_diagram.sql:17]: Addressed 😎

**Review by mateo951 — APPROVED** [review]
### Hi @Favourezeugwa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_diagram.sql (hunk 3 above)
Comment (Favourezeugwa) [reply schema_diagram.sql:37]: LOL, Thank you!

**Pull request merged**